api/src/Streams.h

```diff
@@ -73,7 +75,12 @@ class Stream {
 
   inline bool GetNSFW() const {
     boost::shared_lock<boost::shared_mutex> read_lock(lock_);
-    return nsfw_;
+    return service_nsfw_ ? service_nsfw_ : nsfw_;
```

Comment:
:thinking:

Reply:
`service_nsfw_ || nsfw_` maybe?

Reply:
this needs to be updated https://github.com/MemeLabs/Rustla2/blob/master/api/src/Streams.cpp#L15 and here https://github.com/MemeLabs/Rustla2/blob/master/api/src/Streams.cpp#L61